dev/components/menu/tpl.pug

```diff
@@ -9,7 +9,7 @@
     ng-class="{active: vm.url === '/conversations'}")
     i.material-icons message
     span Conversations
-    .unreaded-message(ng-if="vm.unreadMessage") {{ vm.unreadedMessage }}
```

Comment:
Correct form is "unread"
https://www.usingenglish.com/reference/irregular-verbs/read.html